### PR TITLE
Modify list identifiers tests to test resumption [ch112]

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
Added tests surrounding resumption token

### CODE REVIEW

**Comment by ConnorSheremeta** [comment]
This PR contains changes made from this PR: https://github.com/ualbertalib/jupiter/pull/1377

File: test/integration/oaisys_list_identifiers_test.rb [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+                          owner_id: users(:admin).id, title: 'Fancy thesis 2',
+                          dissertant: 'Jane Doe',
+                          language: CONTROLLED_VOCABULARIES[:language].english,
+                          graduation_date: 'Fall 2017')
+                     .tap do |uo|
+      uo.add_to_path(@community.id, @collection2.id)
+      uo.save!
+    end
+
+    @items = 200.times.map do |i|
```
Comment (murny) [inline test/integration/oaisys_list_identifiers_test.rb]: Just curious why we need so many items? How slow is these tests? (i just hope they not being created in fedora 😅 )

Can we accomplish writing the same tests with less items?

Comment (ConnorSheremeta) [reply test/integration/oaisys_list_identifiers_test.rb]: Correct, it is to test pagination. It takes about 2.5 seconds to create these 200 items which is somewhat significant, I'll work on a way to set the per_page variable

Comment (ConnorSheremeta) [reply test/integration/oaisys_list_identifiers_test.rb]: done

Comment (murny) [reply test/integration/oaisys_list_identifiers_test.rb]: Very nice work 👍

File: test/integration/oaisys_list_identifiers_test.rb [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+  end
+
+  def test_list_identifiers_items_xml
+    get oaisys_path(verb: 'ListIdentifiers', metadataPrefix: 'oai_dc'), headers: { 'Accept' => 'application/xml' }
+    assert_response :success
+
+    schema = Nokogiri::XML::Schema(File.open(file_fixture('OAI-PMH.xsd')))
+    document = Nokogiri::XML(@response.body)
+    assert_empty schema.validate(document)
+    puts @response.body
```
Comment (murny) [inline test/integration/oaisys_list_identifiers_test.rb]: We can probably remove this 😄

Comment (mbarnett) [reply test/integration/oaisys_list_identifiers_test.rb]: Gooooood call

Comment (ConnorSheremeta) [reply test/integration/oaisys_list_identifiers_test.rb]: Oops! I'm happy you caught this!

**Comment by ConnorSheremeta** [comment]
> Same feedback as #1377. Code looks good.
> 
> And will need to fix https://github.com/ualbertalib/jupiter/pull/1377/files#r364903074 to get TravisCi working again

Posting this here as well just to have it in both applicable commits: CI isn't passing because Oaisys isn't currently defined in the Gemfile. It will be defined once https://github.com/ualbertalib/jupiter/pull/1361 is merged. However, this tests will still fail until the two open PR's inside Oaisys are merged and the most recent commit is referenced in the gemfile

**Pull request merged**